src/main/cpp/auton/drivePrimitives/DrivePathPlanner.cpp

```diff
@@ -70,6 +71,12 @@ void DrivePathPlanner::Init(PrimitiveParams *params)
     auto speed = m_chassis->GetChassisSpeeds();
 
     auto path = PathPlannerPath::fromPathFile(m_pathname);
+
+    if (FMSData::GetInstance()->GetAllianceColor() == frc::DriverStation::Alliance::kRed)
+    {
+        path.get()->flipPath();
```

Comment:
This returns a new path that we have to set, so you could do something like
path.reset(path.get().flipPath());